### PR TITLE
Modular predictions

### PR DESCRIPTION
**Pull request by KirbyBroderick**

### CODE REVIEW

**Comment by KirbyBroderick** [comment]
Can we test this with a multi-step inference config file?
Also, what do we think about the docstring for the output of multiple-return functions?

File: bin/predictions.py [hunk 1]
```diff
@@ -93,8 +108,67 @@
...
+        print(f.read())
+
+    return config, dask_cluster_script, run_id, sankey
+
+
+def load_bulks_and_filter():
+    pass
+
+
+def load_adsorbates_and_filter():
```
Comment (brookwander) [inline bin/predictions.py]: ?

File: bin/predictions.py [hunk 2]
```diff
@@ -35,31 +35,46 @@
 from catlas.sankey.sankey_utils import Sankey
 
 
-# Load inputs and define global vars
-if __name__ == "__main__":
-    """Run predictions according to input config file
+def parse_inputs():
```
Comment (brookwander) [inline bin/predictions.py:38]: I think we should move all of these upfront function definitions to another util file

**Review by brookwander — CHANGES_REQUESTED** [review]
I think it would be nice to bundle up the last step sankey stuff too. Let me know if you want me to take care of that. Otherwise just the one easy fix I mentioned and LGTM

**Pull request merged**